cibuild.sh

```diff
@@ -14,6 +14,8 @@ usage()
 }
 
 restoreBuildTools(){
+    # Work around issue where NuGet can't restore  because it can't get a lock
+
```

Comment:
By placing TEMP under WORKSPACE, then we know that Jenkins will clean it up.  This at least gives us a clean TEMP folder for each build.